core/frontend/src/tile/map/MapLayerImageryProvider.ts

```diff
@@ -67,7 +67,7 @@ export abstract class MapLayerImageryProvider {
   /** @internal */
   public get status() { return this._status; }
 
-  /** @alpha */
+  /** @beta */
   public get supportsMapFeatureInfo() { return false; }
```

Comment:
```suggestion
  public get supportsMapFeatureInfo(): boolean { return false; }
```

Reply:
Was asked to make this public instead of beta now